certbot/tests/compat/filesystem_test.py

```diff
@@ -6,41 +6,162 @@
 
         # Set ownership of target to authenticated user
         authenticated_user, _, _ = win32security.LookupAccountName("", win32api.GetUserName())
-        security_owner = win32security.GetFileSecurity(target, win32security.OWNER_SECURITY_INFORMATION)
-        security_owner.SetSecurityDescriptorOwner(authenticated_user, False)
-        win32security.SetFileSecurity(target, win32security.OWNER_SECURITY_INFORMATION, security_owner)
+        security_owner = _get_security_owner(probe_path)
+        _set_owner(probe_path, security_owner, authenticated_user)
 
-        filesystem.chmod(target, 0o700)
+        filesystem.chmod(probe_path, 0o700)
 
-        security_dacl = win32security.GetFileSecurity(target, win32security.DACL_SECURITY_INFORMATION)
-        dacl = security_dacl.GetSecurityDescriptorDacl()
+        security_dacl = _get_security_dacl(probe_path)
         # We expect three ACE: one for admins, one for system, and one for the user
-        self.assertEqual(dacl.GetAceCount(), 3)
+        self.assertEqual(security_dacl.GetSecurityDescriptorDacl().GetAceCount(), 3)
 
         # Set ownership of target to Administrators user group
         admin_user = win32security.ConvertStringSidToSid('S-1-5-32-544')
-        security_owner = win32security.GetFileSecurity(target, win32security.OWNER_SECURITY_INFORMATION)
-        security_owner.SetSecurityDescriptorOwner(admin_user, False)
-        win32security.SetFileSecurity(target, win32security.OWNER_SECURITY_INFORMATION, security_owner)
+        security_owner = _get_security_owner(probe_path)
+        _set_owner(probe_path, security_owner, admin_user)
+
+        filesystem.chmod(probe_path, 0o700)
+
+        security_dacl = _get_security_dacl(probe_path)
+        # We expect only two ACE: one for admins, one for system,
+        # since the user is also the admins group
+        self.assertEqual(security_dacl.GetSecurityDescriptorDacl().GetAceCount(), 2)
+
+
+def _get_security_dacl(target):
+    import win32security  # pylint: disable=import-error
+    return win32security.GetFileSecurity(target, win32security.DACL_SECURITY_INFORMATION)
+
+
+def _get_security_owner(target):
+    import win32security  # pylint: disable=import-error
+    return win32security.GetFileSecurity(target, win32security.OWNER_SECURITY_INFORMATION)
+
+
+def _set_owner(target, security_owner, user):
+    import win32security  # pylint: disable=import-error
+    security_owner.SetSecurityDescriptorOwner(user, False)
+    win32security.SetFileSecurity(
+        target, win32security.OWNER_SECURITY_INFORMATION, security_owner)
 
-        filesystem.chmod(target, 0o700)
 
-        security_dacl = win32security.GetFileSecurity(target, win32security.DACL_SECURITY_INFORMATION)
-        dacl = security_dacl.GetSecurityDescriptorDacl()
-        # We expect only two ACE: one for admins, one for system, since the user is also the admins group
-        self.assertEqual(dacl.GetAceCount(), 2)
+def _create_probe(tempdir):
```

Comment:
nit: I think we might as well do this in `setUp` for the test class since each test is being given a temporary directory already and most tests use this file.

Reply:
Done.